widgetssdk/src/main/java/com/glia/widgets/chat/domain/gva/MapGvaGvaQuickRepliesUseCase.kt

```diff
@@ -1,25 +1,21 @@
 package com.glia.widgets.chat.domain.gva
 
-import com.glia.widgets.chat.model.ChatState
-import com.glia.widgets.chat.model.GvaChatItem
+import com.glia.widgets.chat.model.Gva
 import com.glia.widgets.chat.model.GvaQuickReplies
 import com.glia.widgets.core.engagement.domain.model.ChatMessageInternal
 
-internal class MapGvaGvaQuickRepliesUseCase(
-    private val parseGvaButtonsUseCase: ParseGvaButtonsUseCase,
-    private val mapGvaResponseTextUseCase: MapGvaResponseTextUseCase
-) {
-    operator fun invoke(chatMessage: ChatMessageInternal, chatState: ChatState): GvaChatItem {
-        val message = chatMessage.chatMessage
-        val metadata = message.metadata
-
-        if (chatMessage.isHistory && !chatMessage.isLatest) {
-            return mapGvaResponseTextUseCase(chatMessage, chatState)
+internal class MapGvaGvaQuickRepliesUseCase(private val parseGvaButtonsUseCase: ParseGvaButtonsUseCase) {
```

Comment:
Is it okay that `Gva` is repeated twice in the class name?

Reply:
of course not, thanks.

Reply:
fixed - https://github.com/salemove/android-sdk-widgets/commit/fa506584650c3ca6870b22f4570256ea193ea018